infobip_channels/core/http_client.py

```diff
@@ -63,3 +70,31 @@ def get(
         url = self.auth.base_url + endpoint
 
         return requests.get(url=url, headers=headers.dict(by_alias=True), params=params)
+
+    def put(
+        self, endpoint: str, body: Dict, headers: RequestHeaders = None
+    ) -> requests.Response:
+        """Send an HTTP put request to base_url + endpoint.
+
+        :param endpoint: Which endpoint to hit
+        :param headers: Request headers
+        :param body: Body to send with the request
+        :return: Received response
+        """
+        headers = headers or self.put_headers
+        url = self.auth.base_url + endpoint
+
+        return requests.put(url=url, json=body, headers=headers.dict(by_alias=True))
+
+    def delete(
+        self, endpoint: str, headers: RequestHeaders = None
+    ) -> requests.Response:
+        """Send an HTTP delete request to base_url + endpoint.
+
+        :param endpoint: Which endpoint to hit
+        :return: Received response
+        """
+        headers = headers or self.put_headers
```

Comment:
```suggestion
        headers = headers or self.delete_headers
```

Reply:
This was a nice catch, thank you! Top Top